bindings/experimental/distrdf/test/backend/test_dist.py

```diff
@@ -165,17 +166,30 @@ def test_rdataframe_with_treename_and_simple_filename(self):
 
     def test_rdataframe_with_treename_and_filename_with_globbing(self):
         """
-        Check clustered ranges produced when the input dataset is a single ROOT
-        file with globbing.
+        Check clustered ranges produced by the distributed RDataFrame when the
+        input dataset is glob that expands to a single ROOT file in the same
+        folder.
 
         """
         treename = "myTree"
         filename = "2cluste*.root"
         rdf = DistRDataFrameInterface.TestDataFrame(treename, filename)
 
+        # This test is peculiar because we need to provide the full path to the
+        # file in the current working directory of the test. This happens
+        # because of the TChain globbing. That is, the distributed RDataFrame
+        # internally constructs a TChain passing directly the input glob. When
+        # the `build_ranges` method of the dataframe is called, the list of
+        # input files of the internal TChain is retrieved. This triggers the
+        # TChain globbing that expands the glob to an absolute path to the file.
+        # This behaviour is in contrast with an older way of retrieving the list
+        # of input files to a distributed RDataFrame that relied on Python's
+        # glob.glob function and didn't expand to an absolute path, but to a
+        # relative one.
```

Comment:
I would just say `# need absolute paths because TChain globbing expands to full paths`